application/controllers/Update.php

```diff
@@ -10,45 +10,45 @@ class Update extends GO_Controller {
 
   public function updator(){
```

Comment:
This PR looks really good overall!  I'll require a couple days before I have time to pull it down for testing.  I'll advise of any issues at that time.  Thanks,